frontend/appflowy_flutter/lib/plugins/database_view/application/cell/cell_data_persistence.dart

```diff
@@ -48,7 +48,14 @@ class DateCellDataPersistence implements CellDataPersistence<DateCellData> {
     // This is a bit of a hack. This converts the data.date which is in
     // UTC to Local but actually changes the timestamp instead of just
     // changing the isUtc flag
-    final dateTime = DateTime(data.date.year, data.date.month, data.date.day);
+    final dateTime = DateTime(
```

Comment:
Is it the same as just using the `data.date` directly?

Reply:
No not exactly. See the below screenshot
![image](https://user-images.githubusercontent.com/71320345/236098282-1e727da0-debc-473d-a327-298462196831.png)